### PR TITLE
Imapc pop3c improvements

### PR DESCRIPTION
**Pull request by slusarz**

### CODE REVIEW

File: source/admin_manual/mailbox_formats/pop3c.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.. _pop3c_mbox_format:
+
+====================
+Pop3c Mailbox Format
+====================
+
+The pop3c storage accesses a remote POP3 server as if it were a regular
+(local) Dovecot mailbox format. Dovecot can treat it as a dummy storage or
+optionally a more capable storage.
```
Comment (sirainen) [inline source/admin_manual/mailbox_formats/pop3c.rst:9]: I see this is just copied from imapc, but I don't see how pop3c could be treated as anything else than a dummy storage. :) So just remove this sentence?

Also could say that the the POP3 mailbox is visible as the INBOX folder on Dovecot side.

File: source/configuration_manual/mail_location/pop3c/index.rst [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+.. code-block:: none
+
+  pop3c_user = %u
+  pop3c_master_user = masteruser
+  pop3c_password = masteruser-secret
+
+:ref:`Mail user variables <variables-mail_user>` can be used.
+
+See also :ref:`setting-pop3c_password`.
+See also :ref:`setting-pop3c_user`.
```
Comment (sirainen) [inline source/configuration_manual/mail_location/pop3c/index.rst:76]: were we supposed to start using `.. seealso::`?

File: source/configuration_manual/mail_location/pop3c/index.rst [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+See: :ref:`debugging_rawlog`.
+
+
+.. _setting-pop3c_ssl:
+
+``pop3c_ssl``
+-------------
+
+- Default: <empty>
+- Values: <empty>, ``no``, ``pop3s``, ``starttls``
```
Comment (sirainen) [inline source/configuration_manual/mail_location/pop3c/index.rst:137]: Where are you getting empty from? The default is "no", and empty value is invalid:

```
Invalid -o parameter pop3c_ssl=: Invalid value:
```

File: source/configuration_manual/mail_location/pop3c/index.rst [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+.. _setting-pop3c_user:
+
+``pop3c_user``
+--------------
+
+- Default: ``%u``
+- Values: :ref:`string`
+
+The user identity to be used for performing a regular LOGIN to the source POP3
+server.
```
Comment (sirainen) [inline source/configuration_manual/mail_location/pop3c/index.rst:175]: IMAP has LOGIN, but pop3 has USER+PASS. So this should say something else than LOGIN. Same elsewhere in this page.

**Review by sirainen — CHANGES_REQUESTED** [review]

**Comment by slusarz** [comment]
Let's hold off on this until #344 is resolved.  I explicitly did not change any of the pop3c settings generation there, since this MR is pending (and will need to be refactored when/if #344 is approved).

**Comment by slusarz** [comment]
I've implemented all the suggestions.  However ... these changes should be done as part of the general settings conversion that is happening in #344.  The imapc changes is just part of the conversion anyway.  The pop3c changes are a bit more substantial, but doesn't make any sense to keep the old settings and then deal with merge issues, so I've moved pop3c conversion to a separate commit in #344.  Closing this MR.

**Pull request closed without merging**